modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -362,6 +362,34 @@ test.describe('Manage object definitions through Model Builder', () => {
 			objectDefinition2.id
 		);
 	});
+
+	test('delete object via objects card', async ({
```

Comment:
Hey @rafaellajordao could you rename this test to: **can delete an object definition via the object definition node header**?

Reply:
sure! fixed!